opencti-platform/opencti-graphql/src/resolvers/stixRefRelationship.js

```diff
@@ -44,7 +45,8 @@ const stixRefRelationshipResolvers = {
     opinions: (rel, _, context) => opinionsLoader.load(rel.id, context, context.user),
     creators: (rel, _, context) => creatorsLoader.load(rel.creator_id, context, context.user),
     editContext: (rel) => fetchEditContext(rel.id),
-    datable: (rel) => isDatable(rel.fromType, rel.relationship_type)
+    datable: (rel) => isDatable(rel.fromType, rel.relationship_type),
+    spec_version: (rel) => { return rel.spec_version ?? STIX_SPEC_VERSION; }
```

Comment:
Should we extend this behavior to SDO and SRO?

Reply:
We agreed to extend it to SRO (stix core and stix sighting)